### ISSUE TITLE
OK button function in playback

### ISSUE THREAD

**Issue by kuro68k** [yuliskov/SmartTubeNext#199]
The function of the OK button when playing is a bit annoying. It should ideal go to the icon row, but instead you hand to press down every time. There is no point going to the seek bar as you can use that with left/right without pressing OK.

Also when seeking the OK button pauses even when the pause function is disabled in preferences.

**Comment by stagestage** [comment]
+1

**Comment by rodalpho** [comment]
Agree, I would also like the down arrow to go directly to the icon row-- that is the behavior I'm used to from SYTV.
